src/api/index.ts

```diff
@@ -439,12 +439,12 @@ export class AtelierAPI {
     }
   }
 
-  public serverInfo(): Promise<Atelier.Response<Atelier.Content<Atelier.ServerInfo>>> {
+  public serverInfo(checkNs = true): Promise<Atelier.Response<Atelier.Content<Atelier.ServerInfo>>> {
```

Comment:
```suggestion
  public serverInfo(checkNs: boolean = true): Promise<Atelier.Response<Atelier.Content<Atelier.ServerInfo>>> {
```

Reply:
ESLint strips that type annotation out when you run `mpn run lint-fix`.

Reply:
Fair enough.